src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,57 @@
-export const Tabs = () => {};
+import { useEffect } from 'react';
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void
+  selectedTab: Tab
+
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  onTabSelected,
+  selectedTabId,
+  selectedTab,
+}) => {
+  useEffect(() => {
+    const selected = tabs.find(tab => tab.id === selectedTabId);
+
+    onTabSelected(selected || tabs[0]);
+  }, []);
```

Comment:
why do you change the selected tab on mount? you already have default value for `selectedTab` in `App.tsx`
```suggestion
```